lib/elasticity/instance_type_config.rb

```diff
@@ -0,0 +1,30 @@
+module Elasticity
+
+  class InstanceTypeConfig
+
+    attr_accessor :bid_price
+    attr_accessor :type
+    attr_accessor :weighted_capacity
+
+    def initialize
+      @bid_price = nil
+      @type = 'm1.small'
```

Comment:
Is there a reason you have this default? I don't see default values in other places.

Reply:
It came from here https://github.com/sewichi/elasticity/pull/23/files#diff-9f7b3c295437bc17706c651a9ec36840R19, and I do not care about it.